onboarding_project/squad_pantry_app/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from datetime import datetime
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator
+
+# Create your models here.
+
+
+class Dish(models.Model):
+    dish_name = models.CharField(max_length=256, unique=True)
+    DISH_TYPE = (
+        ('NV', 'Non-Vegetarian'),
+        ('V', 'Vegetarian'),
+        ('E', 'Contains Egg'),
+    )
+    dish_type = models.CharField(max_length=2, choices=DISH_TYPE, default=DISH_TYPE[0][0])
+    is_available = models.BooleanField(default=False, blank=False)
```

Comment:
Is `blank=False` required here? What's the default behavior if you do not specify this?

Reply:
It's getting submitted even if its blank. So, I added `blank=False`.
But it's still getting submitted.

Reply:
Why do you think it is still getting submitted? What can we do to solve this?

Reply:
I'll check!

Reply:
Oops! Logical error! It's correct ...

Reply:
What was the logical error? Also, why is the default value `False`? Why not `True`? What is the behavior in the UI? Is it a dropdown? Can the user enter a null value even if they wanted to?